crates/rerun/src/run.rs

```diff
@@ -58,6 +58,10 @@ struct Args {
     #[clap(long)]
     memory_limit: Option<String>,
 
+    /// Whether the Rerun Viewer should persist the state of the viewer to local storage
```

Comment:
```suggestion
    /// Whether the Rerun Viewer should persist the state of the viewer to disk.
```

This would also be a good place to document where the state is saved, which you can get from https://docs.rs/eframe/latest/eframe/trait.App.html#method.save